geometry/rotation.hpp

```diff
@@ -20,16 +20,52 @@ template<typename FromFrame, typename ToFrame>
 std::ostream& operator<<(std::ostream& out,
                          Rotation<FromFrame, ToFrame> const& rotation);
 
+// |EulerAngles| and |CardanAngles| have values in binary-coded ternary
+// representing the sequence of rotation axes.
+
+enum class EulerAngles {
+  // |ZXZ| is The most common convention, e.g. orbital elements (Ω, i, ω),
+  // rotational elements (90˚ + α₀, 90˚ - δ₀, W).
+  ZXZ = 0b10'00'10,
```

Comment:
I'm not sure what MSVC thinks, but not according to clang or GCC; `0b0'10'00'10` would be an option.
Amusingly `0'10` appears to be a valid literal equal to `8` in both clang and GCC, but`0x` and `0b` don't seem to work similarly.